pkg/metadata/inventories_collector.go

```diff
@@ -50,11 +51,29 @@ func (c inventoriesCollector) Send(ctx context.Context, s serializer.MetricSeria
 	return nil
 }
 
+// getMinInterval gets the inventories_min_interval value, applying the default if it is zero.
+func getMinInterval() time.Duration {
```

Comment:
I think we can leave these in the `config` package since it otherwise breaks isolation of duties without much advantage afaict.

Reply:
I was trying to reduce the `config` API as this logic is very much specific to inventories. I think having the default in config make sens but the logic around enforcing limits in inventory make sens